.gitignore

```diff
@@ -10,6 +10,8 @@ ocs-osd-deployer
 
 # Generated bundle and manifests
 bundle
+bundle.Dockerfile
```

Comment:
Like `bundle/`, operator-sdk generates `bundle.Dockerfile`. I therefore .gitignore it.

Reply:
ack